### ISSUE TITLE
Specify lemma for recorded phrases

### ISSUE THREAD

**Issue by eddieantonio** [UAlbertaALTLab/recording-validation-interface#39]
Generally, some forms in the validation app do appear as expected as audio snippets in itwêwina, whereas other do not, even though one would expect so.

Some examples of not entirely systematic matching of recording snippet transcriptions and normative lemmas generated according to the CW-based FST:

1. Accessible in validation app, but not at all in itwêwina:

kostâcinâkosiw

2. Accessible as inflected form in validation app, and analyzed properly in itwêwina, but not provided as a listed inflected audio form in itwêwina:

kisêhkimaw (<- sêkimêw) [field transcription has aspiration, whereas CW does not]

3. Accessible as inflected form in validation app, and providing linguistic analysis but not providing match with lexical entry (whether in MD or CW) (nor provided as an inflected audio form):

kikostihk (<- kostêw)
kikostew (<- kostêw)

4. Accessible as inflected form in validation app, and matching correct lexical entry in itwêwina, and listed correctly among inflected audio forms.

kikostên (<- kostam)

Very desirable behavior (4) illustrated in screen shot below:

![image](https://user-images.githubusercontent.com/15003407/50144570-ce289380-026c-11e9-8a75-eeeb4a7b8d1c.png)

CONCLUSION: Validation app needs to allow for the specification of the lemma of any recorded form, so that inflected audio forms can be grouped and presented together under the same lemma. Suggestions for lemmas can be provided the descriptive FST.

_Originally posted by @aarppe in https://github.com/UAlbertaALTLab/recording-validation-interface/issues/37#issuecomment-448155202_

**Comment by aarppe** [comment]
(This is discussed in #17) I'd keep 1) words (single orthographical units according to SRO), 2) phrases (2-3 word sequences that are used to convey single-word English meaning, e.g. blind -> 'namoya wâpiw') and 3) full example sentences as separate categories.

Categories 1-2 will likely become new lexical entries (and probably can be treated similarly in validation), whereas category 3 will be linked to all the lexical entries associated with the (mostly) lemmas of the individual words in the sentence.

Nevertheless, the distinction between 2 and 3 is known to be fuzzy (and one cannot rely on the double-colon notation categorically defining 3 not 2).

**Comment by nienna73** [comment]
@aarppe @eddieantonio This issue has been quiet for almost a year. Have parts of it been addressed already?

**Comment by aarppe** [comment]
Nope. We knew this had to be done, and the validation interface (making use of the linguistic analysis of the transcriptions) will enable this to be completed. So, I labeled this as a metaissue for now.
